src/util/rational.cpp

```diff
@@ -10,6 +10,7 @@ Author: Daniel Kroening, kroening@kroening.com
 /// Rational Numbers
 
 #include "rational.h"
+#include "invariant.h"
```

Comment:
Maintain lexicographic ordering of includes: move one line up.

Reply:
The first was the source file's own header. 
Amended to:
```
#include "rational.h"

#include <algorithm>
#include <ostream>

#include "invariant.h"
```